panel/template/golden/golden.html

```diff
@@ -0,0 +1,79 @@
+{% extends base %}
+
+<!-- goes in body -->
+{% block postamble %}
+<script type="text/javascript" src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+<script type="text/javascript" src="https://golden-layout.com/files/latest/js/goldenlayout.js"></script>
+<link type="text/css" rel="stylesheet" href="https://golden-layout.com/files/latest/css/goldenlayout-base.css" />
+<link type="text/css" rel="stylesheet" href="https://golden-layout.com/files/latest/css/goldenlayout-light-theme.css" />
+<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Material+Icons&display=block" rel="stylesheet">
```

Comment:
Would be great if it was possible to specify something else than online resources for examples files directly served by a custom local server or in a static directory

Reply:
For sure, already been thinking of bundling the JS and CSS files in some form.